ui/feed_mark_as_read.go

```diff
@@ -33,5 +33,10 @@ func (h *handler) markFeedAsRead(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	html.Redirect(w, r, route.Path(h.router, "feeds"))
+	redirectToSelf := request.QueryStringParam(r, "redirect_to_self", "false")
```

Comment:
There is also [HasQueryParam()](https://github.com/miniflux/v2/blob/master/http/request/params.go#L98) if you don't want to bother with a default value. That's fine...

Reply:
Duh. Thanks! That being said, I think I don't need to mess at all with this function if I go down the way pointed out in the comment above